CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 3.1.1
```

Comment:
I'd argue for 3.2 instead of 3.1.1

Reply:
Per our conversation we will leave as 3.1.1 - this is a bug fix for non-OIDC flows.